xilinx_test/CMakeLists.txt

```
@@ -123,12 +126,16 @@ endfunction()
 
 synthesis_target("Reduce" "")
 synthesis_target("MultiStageAdd" "")
-sdaccel_target("MultiStageAdd" "")
+sdaccel_target("MultiStageAdd" "" "")
 synthesis_target("AccumulateFloat" "-DHLSLIB_COMPILE_ACCUMULATE_FLOAT")
-sdaccel_target("AccumulateFloat" "-DHLSLIB_COMPILE_ACCUMULATE_FLOAT")
+sdaccel_target("AccumulateFloat" "-DHLSLIB_COMPILE_ACCUMULATE_FLOAT" "")
 synthesis_target("AccumulateInt" "-DHLSLIB_COMPILE_ACCUMULATE_INT")
-sdaccel_target("AccumulateInt" "-DHLSLIB_COMPILE_ACCUMULATE_INT")
+sdaccel_target("AccumulateInt" "-DHLSLIB_COMPILE_ACCUMULATE_INT" "")
 synthesis_target("ShiftRegister" "")
-sdaccel_target("ShiftRegister" "")
+sdaccel_target("ShiftRegister" "" "")
 synthesis_target("StreamAPI" "")
 synthesis_target("Subflow" "")
+set(hbmcfgargs "--config;${CMAKE_CURRENT_SOURCE_DIR}/configs/hbmkernel.cfg")
+set(ddrcfgargs "--config;${CMAKE_CURRENT_SOURCE_DIR}/configs/ddrkernel.cfg")
+sdaccel_target("HBMandBlockCopy" "" "${hbmcfgargs}")
+sdaccel_target("DDRExplicit" "" "${ddrcfgargs}") 
```

The convention for CMake macros is `ALL_UPPER_CASE`:
```suggestion
set(HBM_CONFIG_ARGS "--config;${CMAKE_CURRENT_SOURCE_DIR}/configs/hbmkernel.cfg")
set(DDR_CONFIG_ARGS "--config;${CMAKE_CURRENT_SOURCE_DIR}/configs/ddrkernel.cfg")
sdaccel_target("HBMandBlockCopy" "" "${HBM_CONFIG_ARGS}")
sdaccel_target("DDRExplicit" "" "${DDR_CONFIG_ARGS}") 
```